src/components/UNSTABLE_Pagination/Pagination.stories.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file Unstable pagination stories.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { UNSTABLE_Pagination, PageSelector } from '../..';
+
+import { components } from '../../../.storybook';
+
+storiesOf(components('UNSTABLE Pagination'), module)
+  .addDecorator(story => <div style={{ width: '800px' }}>{story()}</div>)
+  .add(
+    'default',
+    () => (
+      <UNSTABLE_Pagination totalItems={350} pageSizes={[10, 15, 20, 25]}>
+        {({ currentPage, onSetPage, totalPages }) => (
+          <PageSelector
+            id="select-1"
+            onChange={event => onSetPage(event.target.value)}
+            totalPages={totalPages}
+            value={currentPage}
+          />
+        )}
+      </UNSTABLE_Pagination>
+    ),
+    {
+      info: {
+        propTables: [UNSTABLE_Pagination, PageSelector],
```

Comment:
@SimonFinney I found a way to specify which proptables are shown, in situations where a component (in this case, `PageSelector` is being used as a render prop, etc, in such a way that Storybook doesn't detect its usage when generating prop tables 👀 ) -- https://www.npmjs.com/package/@storybook/addon-info#options-and-defaults

Reply:
Very cool - nice find!